### PR TITLE
Improve the accessibility of the top menu in ISIS part 2

### PR DESCRIPTION
**Pull request by brianteeman**
Two links contains no text.
**Why It Matters**
If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users.

This PR adds text for the links for screenreaders on the joomla logo top left and the user menu top right
#### Testing Instructions

Review generated code
or
Review using http://wave.webaim.org/ or its chrome extension
##### Further technical information

http://webaim.org/standards/wcag/checklist#sc2.4.4

### CODE REVIEW

File: administrator/templates/isis/index.php [hunk 1]
```diff
@@ -203,7 +208,11 @@ function colorIsLight($color)
 				<ul class="nav nav-user<?php echo ($this->direction == 'rtl') ? ' pull-left' : ' pull-right'; ?>">
 					<li class="dropdown">
 						<a class="<?php echo ($hidden ? ' disabled' : 'dropdown-toggle'); ?>" data-toggle="<?php echo ($hidden ? '' : 'dropdown'); ?>" <?php echo ($hidden ? '' : 'href="#"'); ?>><span class="icon-user"></span>
-							<span class="caret"></span></a>
+							<span class="caret"></span>
+							<div class="element-invisible">
+							<?php echo JText::_('TPL_ISIS_USERMENU'); ?>
```
Comment (zero-24) [inline administrator/templates/isis/index.php]: can we have one more tab here?

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 4c8144bacba0a2bf9c2c0980f8e3c14e300453ae<br /><br />Thank you<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11729">issues.joomla.org/joomla-cms/11729</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 4c8144bacba0a2bf9c2c0980f8e3c14e300453ae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11729">issues.joomla.org/joomla-cms/11729</a>.</sub>

**Comment by zero-24** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11729">issues.joomla.org/joomla-cms/11729</a>.</sub>

**Comment by brianteeman** [comment]
Thansk

**Pull request merged**